docs/doc/80-releases/95-v1.1.0.md

```diff
@@ -0,0 +1,45 @@
+---
+title: What's new in the v1.1 series release
+sidebar_label: v1.1
+description:
+  Release notes for v1.1 series of Databend.
+---
+
+Databend v1.0 was officially released on April 14, 2023! 
+Thanks to all the community partners who participated and to everyone who made Databend better!
+
+To read more about this release, please refer to our detailed [blog post](https://databend.rs/blog/databend-release-v1.1). You can also find the full changelog of this release on [GitHub](https://github.com/datafuselabs/databend/releases/tag/v1.1.0-nightly).
```

Comment:
To learn more about ...

Reply:
To learn more about this release, please refer to this [blog post](https://databend.rs/blog/databend-release-v1.1) for details.